build.gradle

```diff
@@ -71,3 +72,7 @@ subprojects {
 
     apply from: '../gradle/publish.gradle'
 }
+
+configure(subprojects.findAll {it.name != 'jaeger-generated'}) {
+   apply plugin: 'net.ltgt.errorprone'
```

Comment:
Looking at the generated files, all of them have something like the following in them
`@Generated(value = "Autogenerated by Thrift Compiler (0.9.3)", date = "2016-09-21")` 
I'll look into whether we can make error prone respect this. Their [documentation](http://errorprone.info/docs/flags) doesn't show anything promising.